### PR TITLE
Improve wording around Hive's config management in docs

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: docs/architecture.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Once these steps are completed, an install pod will be launched. The install pod
 
 In the event of an install failure, Hive will cleanup any cloud resources created and keep trying indefinitely (with backoff).
 
-Once the install completes successfully, the admin password and kubeconfig will be uploaded as secrets and linked to the ClusterDeployment. Controllers related to day 2 configuration management now begin reconciling to apply Kubernetes configuration to the end cluster itself (predominantly via the [SyncSet](syncset.md) CRD and controller).
+Once the install completes successfully, the admin password and kubeconfig will be uploaded as secrets and linked to the ClusterDeployment. Controllers related to configuration management now begin reconciling to apply Kubernetes configuration to the end cluster itself (predominantly via the [SyncSet](syncset.md) CRD and controller).
```
[the remark this answers is not in the document]
Comment (LalatenduMohanty) [reply docs/architecture.md]: IMO in architecture document we should say `config management`  as it explains the behavior of controllers at this point.

File: docs/using-hive.md [hunk 2]
```diff
@@ -173,7 +173,7 @@ spec:
 You can now request clusters with clusterDeployment.spec.manageDNS=true and clusterDeployment.spec.baseDomain=mycluster.hive1.example.com. Hive will create the mycluster.hive1.example.com DNS zone, and the OpenShift installer will create DNS entries such as api.mycluser.hive1.example.com.
 
 
-## Day 2 Configuration Management
+## Configuration Management
```
[the remark this answers is not in the document]
Comment (LalatenduMohanty) [reply docs/using-hive.md]: I agree.

File: README.md [hunk 3]
```diff
@@ -3,10 +3,9 @@
 API driven OpenShift 4 cluster provisioning and management.
 
 Hive is an operator which runs as a service on top of Kubernetes/OpenShift.
-The Hive service can be used to provision and manage OpenShift clusters.
+The Hive service can be used to provision and perform initial configuration required for fresh installed OpenShift clusters.
```
Comment (dgoodwin) [inline README.md]: I think "initial configuration" implies "fresh installed", I would just go with "and perform initial configuration of OpenShift clusters". That or s/fresh/freshly.

**Pull request merged**